src/main/java/com/cse/helium/app/dao/ActorsDao.java

```diff
@@ -61,23 +62,32 @@ public ActorsDao(IConfigurationService configService) {
    * @return Flux/<T/> is returned to contains results for the specific entity type
    */
   public Flux<?> getAll(Map<String, Object> queryParams, Integer pageNumber, Integer pageSize) {
-    String contains = "";
     String query = null;
 
     if (queryParams.containsKey("q")) {
       query = queryParams.get("q").toString();
     }
 
+    final String actorDocumentType = Constants.ACTOR_DOCUMENT_TYPE;
+    final SqlQuerySpec actorQuerySpec = new SqlQuerySpec();
+
+    SqlParameterList parameterList = new SqlParameterList();
+    parameterList.add(new SqlParameter("@type", actorDocumentType));
```

Comment:
Can you just use Constants.actor_document_type here and avoid the variable?

Reply:
Fixed!